resources/queue_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021 Nitric Technologies Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resources
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/golang/mock/gomock"
+
+	nitricv1 "github.com/nitrictech/apis/go/nitric/v1"
+	mock_v1 "github.com/nitrictech/go-sdk/mocks"
+	"github.com/nitrictech/go-sdk/mocks/mockapi"
+)
+
+var _ = Describe("queue", func() {
+	ctrl := gomock.NewController(GinkgoT())
```

Comment:
The controller needs to be finalised so assertions on mocks can be properly evaluated `ctrl.Finish()`

Reply:
```golang
// NewController returns a new Controller. It is the preferred way to create a
// Controller.
//
// New in go1.14+, if you are passing a *testing.T into this function you no
// longer need to call ctrl.Finish() in your test methods.
```